### PR TITLE
Ft 39 use axios mock adapter for content on pages

### PR DESCRIPTION
**Pull request by thatrafi**

### CODE REVIEW

File: src/organisms/pricing/PricingPlan.js [hunk 1]
```diff
@@ -2,64 +2,60 @@ import React, { useState, useEffect } from 'react';
...
 import PricingItem from 'molecules/pricing/PricingItem';
 import TabAction from 'molecules/tabAction/TabAction';
-import styles from './PricingPlan.module.scss';
+import styles from './PricingPlan.module.css';
+import axios from 'axios';
+import { filteredArray } from 'helper/ArrayHelper';
 
 const PricingPlan = () => {
-  const [isSelected, setisSelected] = useState(1);
+  const [isSelected, setisSelected] = useState(`plan${0}`);
```
Comment (mylonasg88) [inline src/organisms/pricing/PricingPlan.js]: Isn't it the same as `useState('plan0')`?

File: src/pages/Product/ProductPage.js [hunk 2]
```diff
@@ -8,49 +9,43 @@ import FeatureHome2 from 'organisms/home/FeatureHome2';
...
+    getData();
+  }, []);
+  const getData = async () => {
+    const [{ data: features }, { data: infographics }] = await Promise.all([
+      axios.get('/api/product/features'),
+      axios.get('/api/product/infographics')
+    ]);
+    setFeatures(features);
+    setInfoGraphics(infographics);
+  };
```
Comment (mylonasg88) [inline src/pages/Product/ProductPage.js:26]: Before `return` it is nice to have a space always. Maybe it is possible to add that to the eslint?

File: src/pages/Product/ProductPage.js [hunk 3]
```diff
@@ -8,49 +9,43 @@ import FeatureHome2 from 'organisms/home/FeatureHome2';
...
+  const [infographics, setInfoGraphics] = useState([]);
+  useEffect(() => {
+    getData();
+  }, []);
+  const getData = async () => {
+    const [{ data: features }, { data: infographics }] = await Promise.all([
+      axios.get('/api/product/features'),
+      axios.get('/api/product/infographics')
+    ]);
+    setFeatures(features);
```
Comment (mylonasg88) [inline src/pages/Product/ProductPage.js]: Here an improvement would be to `features && setFeatures(features)`;
Same in the line below. That way if sanity check (whatever that is) is valid then component should set new values which will trigger component re-render.
That way it is always guaranteed that `features` and `infografics` is an Array and validation check below in JSX can be removed :)

**Review by mylonasg88 — CHANGES_REQUESTED** [review]

File: src/organisms/pricing/PricingPlan.js (hunk 1 above)
Comment (thatrafi) [reply src/organisms/pricing/PricingPlan.js]: yesss, i'll remove the `

File: src/pages/Product/ProductPage.js (hunk 2 above)
Comment (thatrafi) [reply src/pages/Product/ProductPage.js:26]: i have fixed it in FT-45

**Pull request merged**